taurus/client/json_encoder.py

```diff
@@ -1,269 +1,53 @@
+# flake8: noqa
 """Encoding and decoding taurus objects to/from json."""
-import json
-from json import JSONEncoder
-
-from taurus.entity.attribute.condition import Condition
-from taurus.entity.attribute.parameter import Parameter
-from taurus.entity.attribute.property import Property
-from taurus.entity.attribute.property_and_conditions import PropertyAndConditions
-from taurus.entity.base_entity import BaseEntity
-from taurus.entity.bounds.categorical_bounds import CategoricalBounds
-from taurus.entity.bounds.composition_bounds import CompositionBounds
-from taurus.entity.bounds.integer_bounds import IntegerBounds
-from taurus.entity.bounds.real_bounds import RealBounds
-from taurus.entity.dict_serializable import DictSerializable
-from taurus.entity.file_link import FileLink
-from taurus.entity.link_by_uid import LinkByUID
-from taurus.entity.object import ProcessRun, MaterialRun, MeasurementRun
-from taurus.entity.object.ingredient_run import IngredientRun
-from taurus.entity.object.ingredient_spec import IngredientSpec
-from taurus.entity.object.material_spec import MaterialSpec
-from taurus.entity.object.measurement_spec import MeasurementSpec
-from taurus.entity.object.process_spec import ProcessSpec
-from taurus.entity.source.performed_source import PerformedSource
-from taurus.entity.template.condition_template import ConditionTemplate
-from taurus.entity.template.material_template import MaterialTemplate
-from taurus.entity.template.measurement_template import MeasurementTemplate
-from taurus.entity.template.parameter_template import ParameterTemplate
-from taurus.entity.template.process_template import ProcessTemplate
-from taurus.entity.template.property_template import PropertyTemplate
-from taurus.entity.value.discrete_categorical import DiscreteCategorical
-from taurus.entity.value.empirical_formula import EmpiricalFormula
-from taurus.entity.value.nominal_categorical import NominalCategorical
-from taurus.entity.value.nominal_composition import NominalComposition
-from taurus.entity.value.nominal_integer import NominalInteger
-from taurus.entity.value.nominal_real import NominalReal
-from taurus.entity.value.normal_real import NormalReal
-from taurus.entity.value.uniform_integer import UniformInteger
-from taurus.entity.value.uniform_real import UniformReal
-from taurus.enumeration.base_enumeration import BaseEnumeration
-from taurus.util import flatten, substitute_links, set_uuids
+import deprecation
```

Comment:
Is the inclusion of `deprecation` conflating a separate issue into this proposal?

Reply:
We moved these methods into `taurus.json` so this just deprecates their old location.  `taurus.client.json_encoder` was a vestige from much earlier development (when I had envisioned implementing a whole local client).